src/components/Transactions/TransactionDetail.tsx

```diff
@@ -85,7 +82,11 @@ export default function TransactionDetail({
                         {t('app.transactionDetail.transactionId')}
                       </Typography>
                       <Typography fontWeight={600} color={theme.palette.primary.main}>
-                        <u>{transactionData.transactionId.substring(0, 20)}...</u>
+                        <u>
```

Comment:
The "underline" is just a cosmetic issue. I suppose it's better to use css instead a specific html tag as `u` 